Userland/Services/AudioServer/main.cpp

```diff
@@ -19,7 +19,12 @@ ErrorOr<int> serenity_main(Main::Arguments)
 
     auto config = TRY(Core::ConfigFile::open_for_app("Audio", Core::ConfigFile::AllowWriting::Yes));
     TRY(Core::System::unveil(config->filename(), "rwc"sv));
-    TRY(Core::System::unveil("/dev/audio", "wc"));
+
+    auto audio_unveil_result = Core::System::unveil("/dev/audio", "wc");
+    // System may not have audio devices, which we handle gracefully.
+    if (audio_unveil_result.is_error())
+        dbgln("Couldn't unveil audio devices: {}", audio_unveil_result.error());
```

Comment:
Instead of making all classes know about the "there's actually no device" case, should this just spin a dummy event loop that drops all events on the floor?

Reply:
that's kind of what we do anyways? and if we don't reply to certain messages, apps still break.

Reply:
also, only one class knows; Mixer